addons/account/static/src/js/reconciliation/reconciliation_action.js

```diff
@@ -252,12 +252,13 @@ var StatementAction = AbstractAction.extend({
      *
      * @private
      */
-    _openFirstLine: function () {
+    _openFirstLine: function (previous_handle) {
         var self = this;
-
-        var handle = _.compact(_.map(this.model.lines,  function (line, handle) {
-                return line.reconciled ? null : handle;
-            }))[0];
+        var handle = (previous_handle && _.compact(_.map(this.model.lines,  function (line, handle) {
+            return (line.reconciled || (parseInt(handle.substr(5)) < parseInt(previous_handle.substr(5)))) ? null : handle;
```

Comment:
not really readable and if nothing is found in first condition, we will redo it again, It would be better to have smth like this, also I'm not sure I understand the point of doing a substr and parseInt to compare as we can directly compare string and it should work
```javascript
if (!previous_handle) previous_handle = ''
var handle = _compact(_.map(this.model.lines, function (line, handle) {
     return (line.reconciled || handle < previous_handle) ? null : handle
}))[0];
...
```

Reply:
You need to use `parseInt` because:
* For strings: `'2' > '10'`
* For numbers: `2 < 10`

I redid the whole loop twice because if we didn't find anything in the first, I wanted to still open the first line. But I guess we could do without it...